Apps/W1/HybridGP/app/src/pages/GPCompanyMigrationSettings.Page.al

```diff
@@ -0,0 +1,155 @@
+page 40056 "GP Company Migration Settings"
+{
+    ApplicationArea = All;
+    Caption = 'GP Company Migration Settings';
+    PageType = List;
+    SourceTable = "GP Company Additional Settings";
+    SourceTableView = sorting(Name) where("Name" = filter(<> ''));
+    UsageCategory = Lists;
```

Comment:
Minor - Worksheet is usually good for this kind of tables that should always be editable.

Reply:
Updated, thanks